### PR TITLE
<drop> hard-code our own bot name

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/cc @kargakis

### CODE REVIEW

File: mungegithub/mungers/path_label.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ var (
 )
 
 const (
-	botName        = "k8s-merge-robot"
+	botName        = "openshift-merge-robot"
 	jenkinsBotName = "k8s-bot"
```
Comment (kargakis) [inline mungegithub/mungers/path_label.go]: Do we also want to update this one?

Comment (kargakis) [inline mungegithub/mungers/path_label.go]: Well, no. We don't use path_labels for now..

**Comment by eparis** [comment]
why not make it a flag/config upstream?

**Comment by stevekuznetsov** [comment]
@eparis tourniquet to stop the bleeding

**Comment by stevekuznetsov** [comment]
later we do surgery

**Pull request merged**